docs/reference/objects/product/promotion.md

```diff
@@ -8,33 +8,50 @@ When using a `promotion` object you have access to the following attributes
 
 # promotion.active
 
-Returns true if the promotion is currently active
+Returns true if the promotion is currently active.
+
+# promotion.category
+
+Returns the category of the promotion.
 
 # promotion.dates
 
-Returns the dates the promotion is active for as a humanized string e.g. `7 - 14 August 2019`
+Returns the dates the promotion is active for as a humanized string e.g. `7 - 14 August 2019`.
 
-# promotion.discount_amount
+# promotion.discount_amount _(deprecated)_
 
-Returns the discount the promotion will apply, if the promotion represents a percentage discount this will be formatted as `25%` if it is a monetary discount this will be as `$25.00` 
+Returns the discount the promotion will apply, if the promotion represents a percentage discount this will be formatted as `25%` if it is a monetary discount this will be as `$25.00`.
 
-# promotion.category
+> **Deprecated**
+ >
+ > This is deprecated. Use `promotion.discount_amount_fractional` or `promotion.discount_percentage` instead. `promotion.is_discount_percentage` can be used to determine which method to use.
+
+# promotion.discount_amount_fractional
 
-Returns the category of the promotion
+Returns the discount the promotion will apply when it's a monetary amount e.g. `12_50`.
```

Comment:
not 12.50?

Reply:
@eballantine good spot. I was trying to represent how the fractional will look like, but I understand is not very clear. Liquid is actually converting the "12_50" into "1250". I'll update the doc with it.